docs/src/main/asciidoc/redis.adoc

```diff
@@ -89,6 +89,20 @@ docker run --ulimit memlock=-1:-1 -it --rm=true --memory-swappiness=0 --name red
 If you use <<dev-services,DevServices>>, launching the container manually is not necessary!
 ====
 
+== Shared server
```

Comment:
Can you extract the dev services for Redis documentation into its own file as we did for Kafka and AMQP.
We are refactoring the documentation to distinguish the "getting started" part from the "reference" part and the configuration of dev services is in the later part.

Reply:
Sure, I'll do that.